dev/ci/util_free_space.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+#
```

Comment:
We can't use this license header. This file is from Apache arrow. We should add license information in License file.

Reply:
Thank you for the reminder. This file is here for testing purposes. I will move it to another PR later and update LICENSE

Reply:
This file is deleted in this PR